Recipes/Recipes.DataAccess/Repositories/GenericRepository.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Recipes.DataAccess.Interfaces;
+
+namespace Recipes.DataAccess.Repositories;
+
+public class GenericRepository<T> : IGenericRepository<T> where T : class
+{
+    private readonly DbContext _dbContext;
+    private readonly DbSet<T> _dbSet;
+
+    public GenericRepository(DbContext dbContext, DbSet<T> dbSet)
+    {
+        _dbContext = dbContext;
+        _dbSet = dbSet;
+    }
+
+    public async Task<T> CreateAsync(T entity)
```

Comment:
Regarding the warnings in GitHub - I think you can just remove the async keywords for now from here until they are implemented.